game/src/test/java/com/hiddenswitch/spellsource/CustomCardsTests.java

```diff
@@ -7135,5 +7135,24 @@ public void testIrenaDragonKnight() {
 		});
 	}
 
+	// Mollusk Meister: "Opener: Gain 8 Armor. Give a friendly minion Health equal to your Armor.",
+	@Test
+	public void testMolluskMeister() {
+		// Empty deck
+		runGym((context, player, opponent) -> {
+			player.getHero().modifyArmor(3);
+			playMinionCard(context, player, "minion_mollusk_meister");
+			assertEquals(player.getHero().getArmor(), 11);
+		});
+		// Non-empty deck
+		runGym((context, player, opponent) -> {
+			player.getHero().modifyArmor(3);
+			Minion friendly = playMinionCard(context, player, "minion_neutral_test");
+			playMinionCardWithBattlecry(context, player, "minion_mollusk_meister", friendly);
```

Comment:
Yeah, so you changed the card the wrong way. Its original code was correct I think. Here, you are coercing it to have a target, which coincidentally works, but it needs to have a target selection of friendly minions. Right now, it is buffing all friendly minions.